install.js

```diff
@@ -123,14 +123,14 @@ if (redirectURL.startsWith("https")) {
 // Create and populate application config file
 console.log("Creating required files...");
 
-let data = `{
-  "token": "${token}",
-  "applicationId": "${appId}",
-  "oauth2Secret": "${appSecret}",
-  "redirectURI": "${domain}",
-  "privateKey": "${privateKey ? privateKey : ""}",
-  "certificate": "${certificate ? privateKey : ""}"
-}`;
+let data = JSON.stringify({
+    token: token,
+    applicationId: appId,
+    oauth2Secret: appSecret,
+    redirectURI: domain,
+    privateKey: privateKey,
+    certificate: certificate
+});
```

Comment:
I would personally add `, null, 4` to the arguments of stringify, so it adds indentation and newlines.
Makes the config easier to read and edit.